terraform/fdi_lambda.tf

```diff
@@ -1,23 +1,9 @@
 # Infrastructure related to the findings data import lambda
 
-# IAM assume role policy document for the roles we're creating for the
-# lambda function
-data "aws_iam_policy_document" "fdi_lambda_assume_role_doc" {
-  statement {
-    effect = "Allow"
-
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["lambda.amazonaws.com"]
-    }
-  }
-}
-
 # The role we're creating for the lambda function
 resource "aws_iam_role" "fdi_lambda_role" {
-  assume_role_policy = data.aws_iam_policy_document.fdi_lambda_assume_role_doc.json
+  name               = format("fdi_lambda_role_%s", local.production_workspace ? "production" : terraform.workspace)
+  assume_role_policy = data.aws_iam_policy_document.lambda_service_assume_role_doc.json
 }
 
 # IAM policy document that that allows some Cloudwatch permissions
```

Comment:
Capitalization - there are probably a bunch of other similar cases if you feel like fixing them:

```suggestion
# IAM policy document that that allows some CloudWatch permissions
```

Reply:
Please see [021f2d9](https://github.com/cisagov/cyhy_amis/pull/410/commits/021f2d999cfd1d358eeec475db917ee0507cd8e2).